### PR TITLE
Redesign

### PR DESCRIPTION
**Pull request by elmariachi111**

### CODE REVIEW

File: src/components/MailchimpSignup.js [hunk 1]
```diff
@@ -1,65 +1,59 @@
...
-} from 'grommet'
-import { Mail } from 'grommet-icons'
 import React, { useEffect, useState } from 'react'
+import emailValidator from 'email-validator'
+import styled from 'styled-components'
+import { Anchor, Box, Button, Paragraph, TextInput } from 'grommet'
+import { MailOption } from 'grommet-icons'
 
-export function MailchimpSignup(props) {
+const StyledButton = styled(Button)`
```
Comment (elmariachi111) [inline src/components/MailchimpSignup.js:7]: please please please, before you start styling everything on your own in pixels, start modifying the grommet theme file so all rhythms stay the same : e.g https://v2.grommet.io/button#button.padding.vertical

File: src/components/PastMeetups.js [hunk 2]
```diff
@@ -1,75 +1,92 @@
...
+  // const meetups = data.graphcms.meetups
+  const meetups = []
 
-  const meetups = data.graphcms.meetups
   return (
-    <Box pad={{ vertical: 'medium' }} direction="row-responsive" wrap={true}>
+    <Box
+      direction="row-responsive"
+      wrap={true}
+      margin={{ horizontal: '-10px', top: 'medium', bottom: 'large' }}
```
Comment (elmariachi111) [inline src/components/PastMeetups.js:56]: use gap if possible and don't try to hack grommet's grid system on your own.

File: src/components/PastMeetups.js [hunk 3]
```diff
@@ -1,75 +1,92 @@
...
-import { graphql, useStaticQuery, Link, navigate } from 'gatsby'
+import React, { useContext } from 'react'
+import { Box, Heading, Text, Image, ResponsiveContext } from 'grommet'
+import { Calendar } from 'grommet-icons'
+import { graphql, useStaticQuery, navigate } from 'gatsby'
+import styled from 'styled-components'
 import Time from './Time'
 
-const colors = theme.global.colors
+const StyledH3 = styled(Heading)`
```
Comment (elmariachi111) [inline src/components/PastMeetups.js:8]: please use the global theme props to style individual headings: https://v2.grommet.io/heading#heading.level

File: src/components/layout.js [hunk 4]
```diff
@@ -38,8 +57,8 @@ const Layout = ({ children }) => {
       onRedirectCallback={onRedirectCallback}
     >
       <Grommet theme={theme} themeMode="dark">
-        <Box background="black">
-          <AppHeader appName="coding earth" />
+        <Box background={topBackground}>
+          <AppHeader appName="coding earth" currentPath={currentPath} />
```
Comment (elmariachi111) [inline src/components/layout.js:61]: not sure what you mean with "currentPath" but you can get the current path by @reach-router on any component: https://reach.tech/router/api/useLocation

File: src/components/PastMeetups.js (hunk 2 above)
Comment (pribeiro89) [reply src/components/PastMeetups.js:56]: This is not a hack and gap is not possible with wrap per their own instructions. :)

Comment (elmariachi111) [reply src/components/PastMeetups.js:56]: well, from the perspective of flexbox that's a hack (that worked well for a long time of responsive grids, I know) -> here's a solution with plain CSS grid and gaps & dummy content: https://github.com/cod1ng-earth/ce-website/commit/8d8e0723a17ca686b504f0d094fc304e0a3e0932#diff-ca4a299972b3611f7cdd7de2d42a7aa6R61

**Pull request merged**